testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksDelete.java

```diff
@@ -79,28 +84,263 @@ public void testDeletionVectors()
 
             assertThat(onDelta().executeQuery("SELECT * FROM default." + tableName))
                     .containsOnly(row(1, 11));
+            assertThat(onTrino().executeQuery("SELECT * FROM delta.default." + tableName))
```

Comment:
Could you please test the feature also in the context of dealing with checkpoints `delta.checkpointInterval` ?

Either here or in `TestDeltaLakeDatabricksCheckpointsCompatibility`